microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -95,61 +95,64 @@ When viewing the DOM Tree, you may find yourself interested in a DOM node that i
 
 1. The `<p>Magritte</p>` node should still be selected in your DOM Tree.  If not, go back to [Scroll into view](#scroll-into-view) and start over.
 
-1. Right-click the `<p>Magritte</p>` node, and then click **Scroll into view**.  Your viewport scrolls back up to display the **Magritte** node.  See [Appendix: Missing options](#appendix-missing-options) if the **Scroll into view** option isn't displayed.
+1. Right-click the `<p>Magritte</p>` node, and then click **Scroll into view**:
 
-![Scroll into view](./index-images/dom-elements-highlighted-scroll-into-view-dropdown.png)
+   ![Scroll into view](./index-images/scroll-into-view-dropdown.png)
+   
+   Your viewport scrolls back up to display the **Magritte** node.  See [Appendix: Missing options](#appendix-missing-options) if the **Scroll into view** option isn't displayed.
 
-### Search for nodes
+
+#### Search for nodes
 
 You can search the DOM Tree by string, CSS selector, or XPath selector.
 
-1. Focus your cursor on the **Elements** tool.
+1. Focus your cursor on the **Elements** tool, such as by clicking the **Elements** tab.
 
 1. Press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  The Search bar opens at the bottom of the DOM Tree.
 
-1. Type **The Moon is a Harsh Mistress**.  The last sentence is highlighted in the DOM Tree.
+1. Type **The Moon is a Harsh Mistress**.  The last sentence is highlighted in the DOM Tree:
 
-![Highlighting the query in the Search bar](./index-images/dom-elements-highlighted-search-nodes-highlight.png)
+   ![Highlighting the query in the Search bar](./index-images/search-nodes-highlight.png)
 
 The Search bar also supports CSS and XPath selectors.
 
 
 <!-- ====================================================================== -->
 ## Edit the DOM
 
-You can edit the DOM on the fly and review how the changes affect the page.
+You can edit the DOM in DevTools and view how the changes affect the page in real time.
 
-### Edit text content
+#### Edit text content
 
 To edit the text content of a node, double-click the content in the DOM Tree.
```

Comment:
```suggestion
To edit the text content of a node, double-click the content in the DOM Tree, as follows:
```
somewhat boilerplate fluff, but it earns its wordcount b/c helps glue flow and confirm that the list expands the high-level instruction lead-in.